src/huggingface_hub/_login.py

```diff
@@ -111,7 +111,7 @@ def logout() -> None:
 ###
 
 
-def interpreter_login() -> None:
+def interpreter_login(token: Optional[str] = None) -> None:
```

Comment:
(same comment as above)

```suggestion
def interpreter_login() -> None:
```